lib/ttfunk/sub_table.rb

```diff
@@ -0,0 +1,20 @@
+require_relative './reader'
+
+module TTFunk
+  class SubTable
+    include Reader
+
+    attr_reader :file, :table_offset
+
+    # set by parse! in derived classes
```

Comment:
This comment is a bit misleading. It's only true for `Index`, others have it either set in constructor (`Dict`) or hardcodec (`Header`).